src/gluonts/model/rotbaum/_preprocess.py

```diff
@@ -336,11 +406,15 @@ def make_features(self, time_series: Dict, starting_index: int) -> List:
             if self.use_feat_static_real
             else []
         )
-        feat_static_cat = (
-            list(time_series["feat_static_cat"])
-            if self.use_feat_static_cat
-            else []
-        )
+        if self.cardinality:
+            feat_static_cat = (
+                self.encode_one_hot_all(time_series["feat_static_cat"])
+                if self.one_hot_encode
+                else list(time_series["feat_static_cat"])
```

Comment:
Shouldn't this be already a list?

Reply:
I was under the impression that it could potentially be a numpy array as well